src/encoded/schemas/matrix_file.json

```diff
@@ -321,9 +323,11 @@
         }
     },
     "file_format_file_extension": {
-        "mtx": "mtx",
         "csv": ".csv.gz",
-        "loom": ".loom"
+        "loom": ".loom",
+        "hdf5": ".h5",
+        "tsv": [".tsv",".txt"],
```

Comment:
This was added, but looks like it wasn't included in any of the inserts to test it
Turns out it messes up the href download url. We will likely update that code at some point, but I think for now we should maybe allow tsv or txt for file_format and give each their own extensions.

Reply:
Okay, I'll update that to two separate entries for file_format_file_extension.

Reply:
Looks good.